external/corert/src/System.Private.CoreLib/shared/System/Reflection/AssemblyNameFormatter.cs

```diff
@@ -64,7 +64,7 @@ public static string ComputeDisplayName(RuntimeAssemblyName a)
 			if (pkt != null)
 			{
 				if (pkt.Length > PUBLIC_KEY_TOKEN_LEN)
-					throw new ArgumentException();
+					throw new ArgumentException("Invalid token length", nameof(a));
```

Comment:
I thought the `external` directory is not subject to the style rules - it makes this code harder to share with a "death by a thousand insignificant changes" - even though I admit some of this code is dumb.

Reply:
For formatting changes yes but there is no way to exclude single file or folder for roslyn analyzers so I went ahead and fixed the issues this way (it should be all one lines so not that painful to compare)

Reply:
or upstream these changes to corert :)